modeldb/modeldb-run.yaml

```diff
@@ -1705,3 +1705,5 @@
         repl: ''
 267694:
     model_dir: mouse_model/mod_files
+2014998:
+    github: 'pull/1'
```

Comment:
Not sure what we did in the past in case we opened a PR in one of the models. Maybe it's okay to let the CI fail until the fix gets merged?
I'm okay though with merging this until the PR of the model gets merged as long as we don't forget to update this line once this happens

Reply:
I merged the PR. So we can remove this.

Reply:
This needs to be `github: 'default'` now to get the changes from GH or since the change is merged to `master` the model is also updated in `ModelDB`?

Reply:
I added comment in the modeldb repo PR but I am sure it won't be automatically updated on ModelDB. So we can continue using GitHub version.

Reply:
Perfect, I rebased the new changes on `master` and updated the description.